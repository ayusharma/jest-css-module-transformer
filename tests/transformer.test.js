import css from './__fixtures__/sample.css';
import scss from './__fixtures__/sample.css';

describe('Jest css module transformer', () => {
    test('css output', () => {
        expect(css).toMatchSnapshot();
    });

    test('scss output', () => {
        expect(scss).toMatchSnapshot();
    });
});
