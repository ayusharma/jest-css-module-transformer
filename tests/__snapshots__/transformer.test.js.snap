// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest css module transformer css output 1`] = `
Object {
  "button": "button",
  "flex": "flex",
  "text": "text",
}
`;

exports[`Jest css module transformer scss output 1`] = `
Object {
  "button": "button",
  "flex": "flex",
  "text": "text",
}
`;
